SDL/Kernels.cuh

```diff
@@ -26,25 +26,22 @@
 #include "PixelTriplet.cuh"
 #include "TrackExtensions.cuh"
 #include "Constants.cuh"
+#include <alpaka/alpaka.hpp>
 
+#include <alpaka/alpaka.hpp>
```

Comment:
Typo? (Header added twice)

Reply:
Good catch, will fix in a cleanup commit now.